src/main/resources/db/migration/20240523055141906__add_unit_of_orderable_uuid_to_stock_card_line_items.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE stock_card_line_items ADD COLUMN unitoforderableid UUID;
```

Comment:
Update [schema](https://github.com/OpenLMIS/openlmis-stockmanagement/tree/master/src/main/resources/schemas) for that entity.

Reply:
udpated